modules/developer_manual/pages/core/apis/ocs/user-sync-api.adoc

```diff
@@ -0,0 +1,93 @@
+= OCS User Sync API
+:toc: right
+:toclevels: 1
+:request-base-path: ocs/v2.php
+
+This endpoint triggers user-sync for a specific user.
```

Comment:
Would be great if you could add a `== Introduction` section as we have in all other documents 😃

Reply:
Sure. Thanks for spotting that.

Reply:
Done.